enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,244 @@
+OpenShift 4.11, container runtime will be migrated from Docker to containerd
+
+## Motivation
+
+In Kubernetes, the container runtime interface has been used to talk to a container runtime. From
+Kubernetes 1.24 onwards dockershim will be removed from kubelet code. At present
+WMCO uses Docker as the default runtime. The goal is to make containerd the default
+runtime and move away from Docker before dockershim has been decoupled from kubelet.
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Make containerd the default container runtime on Windows nodes.
+* When WMCO is upgraded from 5.x.z to 6.x.z, the runtime on all existing Windows nodes will be migrated to
+  containerd.  
+
+### Non-Goals
+
+* Uninstalling Docker runtime is not part of this enhancement.
+* Allowing the user to choose a container runtime is not supported.
+* Downgrade is not supported.
+
+#### Dev Preview -> Tech Preview
+
+N/A
+
+#### Tech Preview -> GA
+
+N/A
+
+## Proposal
+
+To make containerd the default runtime, containerd should be started as a Windows
+service before starting kubelet as Windows service. While starting kubelet, the parameters `container-runtime` and
+`container-runtime-endpoint` should be configured. The remaining WMCO workflow of bringing Windows nodes into the
+Kubernetes cluster has no impact other than the proposed changes. MachineSet and BYOH (Bring Your Own Host) upgrade
+details are discussed in the upgrade section. There is no difference between MachineSet and BYOH on how we enable
+containerd as the default runtime.
+
+Golden Image:
+
+vSphere and BYOH use golden image to create Windows VM. Customers can create a new Windows golden image excluding Docker
+runtime or continue to use the same Windows image with Docker. Once containerd becomes the default runtime, WMCO will
+support Windows golden image with or without Docker.
+
+Containerd Migration plan:
+
+Containerd will be the default runtime in OpenShift 4.11 for Windows. For WMCO development and testing, we plan
+to introduce a WMCO CLI option which allows us to choose the runtime. This CLI option will be used in the
+release of the WMCO community operator for OpenShift 4.10 to allow customers to try out Windows
+nodes with containerd. Customers can use 4.10 community WMCO to create new clusters or upgrade
+existing clusters.
+
+### User Stories
+
+Stories can be found within the [Windows Containers: containerd epic](https://issues.redhat.com/browse/WINC-505)
+
+### Justification
+We have explored containerd, CRI-O, and Mirantis dockershim as an alternate runtime. After looking into business and
+engineering details, we conclude containerd stands out to be the best solution for WMCO.
+
+Advantage of using containerd:
+* Containerd is widely adopted and supported by the open-source community.
+* With minimal effort, we get new features and bug fixes from the community and enterprise.
+* Better CPU performance and memory usage compared with Docker
```

Comment:
Unless you have numbers to back this up, please remove this line. If you do, please link to it. In the absence of data it could be unnecessarily controversial.